srcpkgs/corecollector/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'corecollector'
+pkgname=corecollector
+version=0.3.5
+revision=1
+build_style=meson
+configure_args="-Dcoredump_user=_corecollector -Dcoredump_group=_corecollector"
+conf_files="/etc/corecollector/corecollector.conf"
+hostmakedepends="dmd git pkg-config scdoc"
+depends="gdb"
+short_desc="Coredump collector written in D"
+maintainer="travankor <travankor@tuta.io>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/Cogitri/corecollector"
+distfiles="https://github.com/Cogitri/corecollector/archive/v${version}.tar.gz"
+checksum=0b9e212ffde1b63024744ccb437504a8ca56d091c1bfd193095b52e0c8a70db6
+nocross="only gcc-gdc(unpackaged as of 3/2020) has cross support"
+
+system_accounts="_corecollector"
```

Comment:
Since I would expect to be able to add myself to this group without issue, shouldn't that group name be without an underscore?

Reply:
https://github.com/void-linux/void-packages/blob/master/Manual.md#runtime_account_creation